doc/doxygen/README.md

```diff
@@ -15,7 +15,7 @@ The simplest way to build the documentation is via Docker using the
 container published to the GitHub Container Registry:
 
 ```
-docker run -v `pwd`/../:/src ghcr.io/OpenAssetIO/openassetio-doc-build bash -c 'make -C /src/doc html'
+docker run -v `pwd`/../:/src ghcr.io/openassetio/openassetio-doc-build bash -c 'make -C /src/doc html'
```

Comment:
It's a bit unfortunate we are duplicating the contents of the `container-build` make command here. Is there any way we can just remove this entirely so we don't have a chance of this being out of date? The one in the `makefile` already had this change.